commit 98829fd1ac81c09c01f8788171472f7e06a3e478
Author: chiachin <chiachin@netivism.com.tw>
Date:   Fri Mar 7 17:41:58 2025 +0800

    Refs #42835, Fix registration date validation by using strtotime.

diff --git a/CRM/Event/BAO/Event.php b/CRM/Event/BAO/Event.php
index 929b93f41..8ece52ab8 100644
--- a/CRM/Event/BAO/Event.php
+++ b/CRM/Event/BAO/Event.php
@@ -1809,8 +1809,8 @@ WHERE  ce.loc_block_id = $locBlockId";
 
   static function validRegistrationDate(&$values, $contactID) {
     // make sure that we are between  registration start date and registration end date
-    $startDate = CRM_Utils_Date::unixTime(CRM_Utils_Array::value('registration_start_date', $values));
-    $endDate = CRM_Utils_Date::unixTime(CRM_Utils_Array::value('registration_end_date', $values));
+    $startDate = strtotime(CRM_Utils_Array::value('registration_start_date', $values));
+    $endDate = strtotime(CRM_Utils_Array::value('registration_end_date', $values));
     $now = time();
     $validDate = TRUE;
     if ($startDate && $startDate >= $now) {
