diff --git CRM/Core/Form.php CRM/Core/Form.php
index 5b93c1a..015609a 100644
--- CRM/Core/Form.php
+++ CRM/Core/Form.php
@@ -247,6 +247,8 @@ class CRM_Core_Form extends HTML_QuickForm_Page {
      * This function is just a wrapper, so that we can call all the hook functions
      */
     function mainProcess() {
+        CRM_Utils_Hook::preSave( get_class( $this ),
+                                     $this );
         $this->postProcess( );
 
         CRM_Utils_Hook::postProcess( get_class( $this ),
diff --git a/CRM/Utils/Hook.php b/CRM/Utils/Hook.php
index 1d36042..f4d3312 100644
--- a/CRM/Utils/Hook.php
+++ b/CRM/Utils/Hook.php
@@ -126,6 +126,24 @@ class CRM_Utils_Hook {
                   $config->userHookClass .
                   '::invoke( 2, $formName, $form, $formName, $formName, $formName, \'civicrm_buildForm\' );' );  
     }
+    /** 
+     * This hook is invoked when a CiviCRM form is before submit. If the module has injected
+     * any form elements, this hook should save the values in the database
+     * 
+     * @param string $formName the name of the form
+     * @param object $form     reference to the form object
+     *
+     * @return null the return value is ignored
+     */
+    static function preSave( $formName, &$form ) {
+        $config = CRM_Core_Config::singleton( );
+        require_once( str_replace( '_', DIRECTORY_SEPARATOR, $config->userHookClass ) . '.php' );
+        return
+            eval( 'return ' .
+                  $config->userHookClass .
+                  '::invoke( 2, $formName, $form, $formName, $formName, $formName, \'civicrm_preSave\' );' );
+    }
+
 
     /** 
      * This hook is invoked when a CiviCRM form is submitted. If the module has injected
