diff --git a/sql/civicrm_data.zh_TW.mysql b/sql/civicrm_data.zh_TW.mysql
index ecf9e06..0eb514f 100644
--- a/sql/civicrm_data.zh_TW.mysql
+++ b/sql/civicrm_data.zh_TW.mysql
@@ -4877,20 +4877,20 @@ VALUES
   (@option_group_id_rr, '/(19|20)(\\d{2})-(\\d{1,2})-(\\d{1,2})/', 'date_', NULL, NULL, 1, NULL, 2, NULL, 0, 0, 0, NULL, NULL),
 
 -- email greeting.
-  (@option_group_id_emailGreeting, 'Dear {contact.first_name}',                                                 1, 'Dear {contact.first_name}',                                                 NULL,    1, 1, 1, NULL, 0, 0, 1, NULL, NULL),
-  (@option_group_id_emailGreeting, 'Dear {contact.individual_prefix} {contact.first_name} {contact.last_name}', 2, 'Dear {contact.individual_prefix} {contact.first_name} {contact.last_name}', NULL,    1, 0, 2, NULL, 0, 0, 1, NULL, NULL),
-  (@option_group_id_emailGreeting, 'Dear {contact.individual_prefix} {contact.last_name}',                      3, 'Dear {contact.individual_prefix} {contact.last_name}',                      NULL,    1, 0, 3, NULL, 0, 0, 1, NULL, NULL),
+  (@option_group_id_emailGreeting, '親愛的 {contact.last_name}{contact.first_name}',                                                 1, '親愛的 {contact.last_name}{contact.first_name}',                                                 NULL,    1, 1, 1, NULL, 0, 0, 1, NULL, NULL),
+  (@option_group_id_emailGreeting, '親愛的 {contact.last_name}{contact.first_name}{contact.individual_prefix}', 2, '親愛的 {contact.last_name}{contact.first_name}{contact.individual_prefix}', NULL,    1, 0, 2, NULL, 0, 0, 1, NULL, NULL),
+  (@option_group_id_emailGreeting, '親愛的 {contact.last_name}{contact.individual_prefix}',                      3, '親愛的 {contact.last_name}{contact.individual_prefix}',                      NULL,    1, 0, 3, NULL, 0, 0, 1, NULL, NULL),
   (@option_group_id_emailGreeting, 'Customized',                                                                4, 'Customized',                                                                NULL, 0, 0, 4, NULL, 0, 1, 1, NULL, NULL),
-  (@option_group_id_emailGreeting, 'Dear {contact.household_name}',                                             5, 'Dear {contact.househols_name}',                                             NULL,    2, 1, 5, NULL, 0, 0, 1, NULL, NULL),
+  (@option_group_id_emailGreeting, '親愛的 {contact.household_name}',                                             5, '親愛的 {contact.household_name}',                                             NULL,    2, 1, 5, NULL, 0, 0, 1, NULL, NULL),
 -- postal greeting.
-  (@option_group_id_postalGreeting, 'Dear {contact.first_name}',                                                 1, 'Dear {contact.first_name}',                                                 NULL,    1, 1, 1, NULL, 0, 0, 1, NULL, NULL),
-  (@option_group_id_postalGreeting, 'Dear {contact.individual_prefix} {contact.first_name} {contact.last_name}', 2, 'Dear {contact.individual_prefix} {contact.first_name} {contact.last_name}', NULL,    1, 0, 2, NULL, 0, 0, 1, NULL, NULL),
-  (@option_group_id_postalGreeting, 'Dear {contact.individual_prefix} {contact.last_name}',                      3, 'Dear {contact.individual_prefix} {contact.last_name}',                      NULL,    1, 0, 3, NULL, 0, 0, 1, NULL, NULL),
+  (@option_group_id_postalGreeting, '親愛的 {contact.last_name}{contact.first_name}',                                                 1, '親愛的 {contact.last_name}{contact.first_name}',                                                 NULL,    1, 1, 1, NULL, 0, 0, 1, NULL, NULL),
+  (@option_group_id_postalGreeting, '親愛的 {contact.last_name}{contact.first_name}{contact.individual_prefix}', 2, '親愛的 {contact.last_name}{contact.first_name}{contact.individual_prefix}', NULL,    1, 0, 2, NULL, 0, 0, 1, NULL, NULL),
+  (@option_group_id_postalGreeting, '親愛的 {contact.last_name}{contact.individual_prefix}',                      3, '親愛的 {contact.last_name}{contact.individual_prefix}',                      NULL,    1, 0, 3, NULL, 0, 0, 1, NULL, NULL),
   (@option_group_id_postalGreeting, 'Customized',                                                                4, 'Customized',                                                                NULL, 0, 0, 4, NULL, 0, 1, 1, NULL, NULL),
-  (@option_group_id_postalGreeting, 'Dear {contact.household_name}',                                             5, 'Dear {contact.househols_name}',                                             NULL,    2, 1, 5, NULL, 0, 0, 1, NULL, NULL),
+  (@option_group_id_postalGreeting, '親愛的 {contact.household_name}',                                             5, '親愛的 {contact.household_name}',                                             NULL,    2, 1, 5, NULL, 0, 0, 1, NULL, NULL),
 
 -- addressee
-  (@option_group_id_addressee, '{contact.individual_prefix}{ } {contact.first_name}{ }{contact.middle_name}{ }{contact.last_name}{ }{contact.individual_suffix}',          '1', '}{contact.individual_prefix}{ } {contact.first_name}{ }{contact.middle_name}{ }{contact.last_name}{ }{contact.individual_suffix}',         NULL ,   '1', '1', '1', NULL , '0', '0', '1', NULL , NULL),
+  (@option_group_id_addressee, '{contact.last_name}{contact.first_name}{ }{contact.individual_prefix}{ }{contact.individual_suffix}',          '1', '{contact.last_name}{contact.first_name}{ }{contact.individual_prefix}{ }{contact.individual_suffix}',         NULL ,   '1', '1', '1', NULL , '0', '0', '1', NULL , NULL),
   (@option_group_id_addressee, '{contact.household_name}',    '2', '{contact.household_name}',    NULL ,   '2', '1', '2', NULL , '0', '0', '1', NULL , NULL),
   (@option_group_id_addressee, '{contact.organization_name}', '3', '{contact.organization_name}', NULL ,   '3', '1', '3', NULL , '0', '0', '1', NULL , NULL),
   (@option_group_id_addressee, 'Customized',                  '4', 'Customized',                  NULL ,    0 , '0', '4', NULL , '0', '1', '1', NULL , NULL),
@@ -16343,7 +16343,7 @@ SELECT @domainID := id FROM civicrm_domain where name = 'Default Domain Name';
 INSERT INTO 
      civicrm_preferences(domain_id, contact_id, is_domain, contact_view_options, contact_edit_options, advanced_search_options, user_dashboard_options, address_options, address_format, mailing_format, display_name_format, sort_name_format, address_standardization_provider, address_standardization_userid, address_standardization_url, editor_id, mailing_backend, contact_autocomplete_options )
 VALUES 
-     (@domainID,NULL,1,'123456789101113','1234567891011','1234567891011121315161718','1234578','123456891011','{contact.address_name}\n{contact.street_address}\n{contact.supplemental_address_1}\n{contact.supplemental_address_2}\n{contact.city}{, }{contact.state_province}{ }{contact.postal_code}\n{contact.country}','{contact.addressee}\n{contact.street_address}\n{contact.supplemental_address_1}\n{contact.supplemental_address_2}\n{contact.city}{, }{contact.state_province}{ }{contact.postal_code}\n{contact.country}','{contact.individual_prefix}{ }{contact.first_name}{ }{contact.last_name}{ }{contact.individual_suffix}','{contact.last_name}{, }{contact.first_name}',NULL,NULL,NULL,2,'a:1:{s:15:"outBound_option";s:1:"3";}','12');
+     (@domainID,NULL,1,'123456789101113','1234567891011','1234567891011121315161718','1234578','123456891011','{contact.postal_code}{ }{contact.state_province_name}{contact.street_address}{contact.supplemental_address_1}{contact.supplemental_address_2}','{contact.postal_code}{ }{contact.state_province_name}{contact.street_address}{contact.supplemental_address_1}{contact.supplemental_address_2}','{contact.last_name}{contact.first_name}{ }{contact.individual_prefix}','{contact.last_name}{contact.first_name}',NULL,NULL,NULL,2,'a:1:{s:15:"outBound_option";s:1:"3";}','12');
 
 
 -- mail settings 
